lib/web/join_tokens.go

```diff
@@ -56,13 +57,40 @@ type scriptSettings struct {
 	appURI         string
 }
 
-func (h *Handler) createScriptJoinTokenHandle(w http.ResponseWriter, r *http.Request, params httprouter.Params, ctx *SessionContext) (interface{}, error) {
+// createTokenRequest is the expected request body of
+// the endpoint to create token
+type createTokenRequest struct {
+	Roles types.SystemRoles `json:"roles"`
+}
+
+func (h *Handler) createTokenHandle(w http.ResponseWriter, r *http.Request, params httprouter.Params, ctx *SessionContext) (interface{}, error) {
+	var req createTokenRequest
+	if err := httplib.ReadJSON(r, &req); err != nil {
+		log.WithError(err).Error("error reading body")
```

Comment:
I don't think these log entries are needed, the error should be logged by the caller (handler wrapper).

Reply:
Thanks, I've removed those in https://github.com/gravitational/teleport/pull/11255/commits/cc452e98ad51209f78177c4f55c62e36519aa385

There are other handlers that log errors, I'll open an issue about this